odoo/addons/base/res/res_config.py

```diff
@@ -492,6 +501,16 @@ def default_get(self, fields):
             if self._fields[name].type == 'selection':
                 res[name] = int(res[name])
 
+        # config: get & convert stored ir.config_parameter (or default)
+        for name, icp in classified['config']:
+            value = SuIrConfigParameter.get_param(icp, self._fields[name].default or False)
```

Comment:
Note that `field.default` is either a callable (even when the field parameter is a value) or `None`. I would replace this line by something like
```python
    field = self._fields[name]
    value = SuIrConfigParameter.get_param(icp, field.default(self) if field.default else False)
```

Reply:
:+1:  sure I will do that